app/models/customer.rb

```diff
@@ -1,3 +1,6 @@
 class Customer < ApplicationRecord
+  validates_presence_of :first_name
+  validates_presence_of :last_name
```

Comment:
I believe this should be: 
`validates :first_name, presence: true`
`validates :last_name, presence: true`